source/includes/usage-examples/code-snippets/find.go

```diff
@@ -9,10 +9,24 @@ import (
 
 	"github.com/joho/godotenv"
 	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
 	"go.mongodb.org/mongo-driver/mongo"
 	"go.mongodb.org/mongo-driver/mongo/options"
 )
 
+// start-restaurant-struct
+type Restaurant struct {
+	ID           primitive.ObjectID `bson:"_id"`
```

Comment:
Does Go not automatically translate from _id to ID (allowing us to not specify `bson:"_id"`)? Just asking since I know some drivers make that conversion automatically. If it doesn't feel free to disregard this comment.

Reply:
It's required in Go! When I tested it out for both this ticket and the findOne usage example, leaving it out renders the ID field as a bunch of zeroes.